src/libtriton/arch/x86/x86Semantics.cpp

```diff
@@ -14696,6 +14698,70 @@ namespace triton {
       }
 
 
+      void x86Semantics::vperm2i128_s(triton::arch::Instruction& inst) {
```

Comment:
Instruction `vperm2i128 dst, src1, src2, src3` - an avx2 [instruction](https://www.laruence.com/x86/VPERM2I128.html) that permutes xmm-parts of src1 and src2 registers (both ymm) into dst. Permutation is defined by control byte src3 (imm8). Destination can be completely zeroed or consisted only from one of the srcs according to src3. Hence non-trivial taint handling used here.